### ISSUE TITLE
Crash during startup caused by "LinkageError" with *both* Create and Valkyrien Skies installed

### ISSUE THREAD

**Issue by unilock** [QuiltMC/quilt-loader#342]
The game crashes during startup with only Quilt Loader + QFAPI/QSL + QKL + Create + Valkyrien Skies installed.

```
Caused by: java.lang.LinkageError: loader constraint violation: when resolving interface method 'void org.apache.logging.log4j.Logger.debug(org.apache.logging.log4j.Marker, java.lang.String, java.lang.Object, java.lang.Object)' the class loader org.quiltmc.loader.impl.launch.knot.KnotClassLoader @e25b2fe of the current class, com/tterrag/registrate/AbstractRegistrate, and the class loader 'app' for the method's defining class, org/apache/logging/log4j/Logger, have different Class objects for the type org/apache/logging/log4j/Marker used in the signature (com.tterrag.registrate.AbstractRegistrate is in unnamed module of loader org.quiltmc.loader.impl.launch.knot.KnotClassLoader @e25b2fe, parent loader org.quiltmc.loader.impl.launch.knot.KnotClassLoader$DynamicURLClassLoader @204f30ec; org.apache.logging.log4j.Logger is in unnamed module of loader 'app')
```

The game loads fine with only Create installed or only Valkyrien Skies installed. Only when the two mods are installed concurrently does it crash with the above error.

Likely related to #232, as well as https://github.com/ValkyrienSkies/Valkyrien-Skies-2/issues/257, https://github.com/ValkyrienSkies/Valkyrien-Skies-2/issues/494, https://github.com/ValkyrienSkies/Valkyrien-Skies-2/issues/524... and others.

[`latest.log`](https://gist.github.com/unilock/b9e4f50e1947cd745c3bf6f5b1c28517#file-latest-log)  
[`crash-2023-08-08_12.33.44-client.txt`](https://gist.github.com/unilock/b9e4f50e1947cd745c3bf6f5b1c28517#file-crash-2023-08-08_12-33-44-client-txt)

- Minecraft: 1.19.2
- Quilt Loader: 0.20.0-beta.11
- QFAPI/QSL: [`qfapi-4.0.0-beta.30_qsl-3.0.0-beta.29_fapi-0.76.0_mc-1.19.2.jar`](https://modrinth.com/mod/qsl/version/4.0.0-beta.30+0.76.0-1.19.2)
- QKL: [`quilt-kotlin-libraries-1.0.2kt.1.8.0flk.1.9.0.jar`](https://modrinth.com/mod/qkl/version/1.0.2+kt.1.8.0+flk.1.9.0)
- Create: [`create-fabric-0.5.1-b-build.1089+mc1.19.2.jar`](https://modrinth.com/mod/create-fabric/version/0.5.1-b-build.1089+mc1.19.2)
- Valkyrien Skies: [`valkyrienskies-1192-2.2.0-beta.3.jar`](https://modrinth.com/mod/valkyrien-skies/version/1.19.2-fabric-2.2.0-beta.3)

**Comment by jwaxy** [comment]
I have the same issue with same versions is there any fixes?

**Comment by AlexIIL** [comment]
This should be fixed in `0.20.0-beta.12` - can you retry?

**Comment by unilock** [comment]
`0.20.0-beta.12` appears to be working fine; thanks!
